Gopkg.lock

```diff
@@ -476,6 +915,22 @@
   revision = "3e01752db0189b9157070a0e1668a620f9a85da2"
   version = "v1.0.6"
 
+[[projects]]
+  digest = "1:0a52bcb568386d98f4894575d53ce3e456f56471de6897bb8b9de13c33d9340e"
+  name = "github.com/spf13/pflag"
```

Comment:
Why Azure as a library is requring flag library? 0.o

Reply:
Fixed by regenerating Gopkg.lock file